pci/src/msix.rs

```diff
@@ -511,6 +511,16 @@ impl MsixCap {
         self.pba & 0xffff_fff8
     }
 
+    pub fn table_set_offset(&mut self, addr: u32) {
+        self.table &= 0x7;
```

Comment:
Don't use fourth person pronoun - and the tone of your commit message is too informal; simply say what was changed and why.

Reply:
Ok, will refine the commit message.